### PR TITLE
[INTEL MKL] BF16 exp

### PR DESCRIPTION
**Pull request by Solaryee**
This PR enables exp for bf16. In Eigen, exp functor will take bf16 as input, up convert to fp32, apply computation, and down convert to bf16. No reduction in it, so there should be no accuracy issue.

### CODE REVIEW

File: tensorflow/python/ops/math_ops_test.py [hunk 1]
```diff
@@ -872,5 +872,29 @@ def testConvertToTensorRange(self):
     self.assertAllEqual(values, self.evaluate(tensor))
 
 
+@test_util.run_all_in_graph_and_eager_modes
```
Comment (penpornk) [inline tensorflow/python/ops/math_ops_test.py:875]: There already are tests for `math_ops.exp` in `python/kernel_tests/cwise_ops_unary_test.py`, for example:
https://github.com/tensorflow/tensorflow/blob/a366858377a2942b691c8664c10e142d8bca8452/tensorflow/python/kernel_tests/cwise_ops_unary_test.py#L206

Could you please add the additional test to `testBfloat16Basic` instead?
https://github.com/tensorflow/tensorflow/blob/a366858377a2942b691c8664c10e142d8bca8452/tensorflow/python/kernel_tests/cwise_ops_unary_test.py#L387

**Review by penpornk — CHANGES_REQUESTED** [review]
Thank you for the PR!

File: tensorflow/python/ops/math_ops_test.py (hunk 1 above)
Comment (ShengYang1) [reply tensorflow/python/ops/math_ops_test.py]: Ok, updated

**Review by penpornk — DISMISSED** [review]
Thank you!

**Comment by ShengYang1** [comment]
@penpornk  The tolerance is not large enough to make the UT pass on GPU, so I make it test only for CPU

**Review by penpornk — APPROVED** [review]
Thank you for the fix!

**Pull request merged**